content/en/docs/collector/internal-telemetry.md

```diff
@@ -113,3 +118,148 @@ journalctl | grep otelcol | grep Error
 ```
 
 {{% /tab %}} {{< /tabpane >}}
+
+## Types of internal observability
+
+The OpenTelemetry Collector aims to be an exemplar of observable service by
+clearly exposing its own operational metrics. It also collects host resource
+metrics that can help you understand if problems are caused by a different
+process on the same host. In this section, you can explore the different types
+of observability emitted by the Collector itself.
+
+### Values observable with internal metrics
+
+The Collector emits internal metrics for the following **current values**:
```

Comment:
Could the @open-telemetry/collector-approvers confirm that all of these metrics have been implemented? I did my best to check the repo and/or the metrics emitted by my own collector instance, but the lists definitely require validation. Thanks!